internal/controller/issuers/apply_test.go

```diff
@@ -32,7 +32,7 @@ func Test_serializeApplyIssuerStatus(t *testing.T) {
 	const (
 		expReg   = `^{"kind":"Issuer","apiVersion":"cert-manager.io/v1","metadata":{"name":"foo","namespace":"bar","creationTimestamp":null},"spec":{},"status":{.*}$`
 		expEmpty = `{"kind":"Issuer","apiVersion":"cert-manager.io/v1","metadata":{"name":"foo","namespace":"bar","creationTimestamp":null},"spec":{},"status":{}}`
-		numJobs  = 10000
+		numJobs  = 100
```

Comment:
Question: Why is this value changed? To speed up tests?

Reply:
That is a mistake, it should have stayed at 10000.